mesop/server/server.py

```diff
@@ -513,3 +525,16 @@ def sse_request(
         if decoded_line.startswith(SSE_DATA_PREFIX):
           event_data = json.loads(decoded_line[len(SSE_DATA_PREFIX) :])
           yield event_data
+
+
+def get_static_assets_folder() -> str | None:
+  folder = os.environ.get("MESOP_STATIC_ASSETS_FOLDER", "")
+  if not folder:
+    return None
+
+  if not os.path.isabs(folder):
```

Comment:
I think we should do relative dir paths for the static assets folder (we can consider the project root the dir that contains the mesop app entrypoint file, e.g. main.py) because:

1) it covers the common use case
2) it prevents security issues (e.g. if there's an absolute dir path like "/" because somebody is confused about this, they may end up exposing sensitive files). Also, we should be careful even with a relative dir path that you cannot traverse above the app dir (e.g. "../foo" should be prohibited) because similarly this can be problematic for security reasons